documentation/Deployment-Guide.md

```diff
@@ -0,0 +1,1136 @@
+| **Traffic Manager Name**    | Please provide the name of the trafficmanager certficate custome domain name   |    | NA |
+| **Azure B2C Application ID**   | Enter App Client ID registrered under B2C tenant   |     |   f58xxxxx-xxxx-xxxx-xxxx-f5xxxxxed570   |
+| **B2C ApplicationID DR**    | Enter Second App Client ID registrered under B2C tenant,if you choose Standard/premium Solution | | NA |
+| **B2C Policy**   | Enter B2C SignUp-SignIn policy      |       |   B2C_x_xxxxxx-In  |
+| **B2C Tenant** |  Enter B2C tenant name ex. something.onmicrosoft.com  |      |    xxxxxxxxxx.onmicrosoft.com  |
+| **B2C Scope** | Enter User impersonation scope of B2C tenant |           |  https://xxxxxxxxx.onmicrosoft.com/xxxxxxx   |
+| **B2C ScopeDR** | Enter Second User impersonation scope of B2C tenant,if you choose Standard/Premium Solution |      | NA      |
+| **video Indexer Key** | Enter Video Indexer service key         |        |    11xxxxxxxxxxxxxxxxxxxxxxxxxxxx54   |
+| **keyVault Name** |      Name of the Key Vault      |        | NA   |
+| **keyVault WebApp SecretName** |     Name of the Web App Secret Name      |        | NA     |
+| **keyVault Resource Group** |        Name of the Key Vault Resource Group   |        | NA     |
+| **WebApp Certificate ThumbPrint** |     Value of Web App Certificate Thumb print     |        | NA     |
+
+## 5 Getting Started
+
+Azure Resource Manager allows you to provision your applications using a declarative template. In a single template, you can deploy multiple services along with their dependencies. The template consists of JSON and expressions that you can use to construct values for your deployment. You use the same template to repeatedly deploy your application during every stage of the application lifecycle.
+
+Resource Manager provides a consistent management layer to perform tasks through Azure PowerShell, Azure CLI, Azure portal, REST API, and client SDKs.
+
+Resource manager provides the following feature:
+
+•	Deploy, manage, and monitor all the resources for your solution as a group, rather than handling these resources individually.
+
+•	Repeatedly deploy your solution through the development lifecycle and your resources are deployed in a consistent state.
+
+•	Manage your infrastructure through declarative templates rather than scripts.
+
+•	Define the dependencies between resources so they're deployed in the correct order.
+
+•	Apply access control to all services in your resource group because Role-Based Access Control (RBAC) is natively integrated into the management platform.
+
+•	Apply tags to resources to logically organize all the resources in your subscription.
+
+
+### 5.1 ARM Template Deployment Using Azure Portal
+
+1.	Click the below **Git hub** repo URL.
+
+https://github.com/sysgain/SecurityAndSurveillance/tree/master
+
+2. Select **main-template.json** from **master** branch, click on **Raw** from the top right corner.
```

Comment:
**main-template.json** --> not found in the repo.

Reply:
we have updated the correct file name i.e. maintemplate.json . Please review and close it.